README.md

```diff
@@ -30,7 +30,7 @@ By default, the server has no password and runs on UDP port 27015.
 If you want to start the server on port **25000** with password **hello**:
 
 ```
-docker run -p 25000:27015/udp -e R2_PSW='hello' avivace/ror2server:latest
+docker run -p 25000:25000/udp -e R2_SV_PORT=25000 -e R2_PSW='hello' avivace/ror2server:latest
 ```
 
```

Comment:
Should I add some more examples?
Default ports with Heartbeat:
```
docker run -p 27015:27015/udp -p 27016:27016/udp -e R2_HEARTBEAT=1 avivace/ror2server:latest
```

Alternate ports with Heartbeat:
```
docker run -p 25000:25000/udp -p 25001 :25001 /udp -e R2_SV_PORT=25000 -e R2_QUERY_PORT=25001 -e R2_HEARTBEAT=1 avivace/ror2server:latest
```

Reply:
@avivace I am requesting a formal opinion from Dr. README.

Reply:
I think this is enough for now, maybe later we can add another section with more example commands. I don't want the readme to be overwhelming just to present a niche use case as the changing everyport for hosting more servers on the same ip